scripts/debug_tools/renew_info_files.py

```diff
@@ -0,0 +1,130 @@
+            return False
+        for f in os.listdir(include_dir):
+            if f.endswith("intrin.h"):
+                return True
+        return False
+
+    result = []
+    for include_dir in include_dirs:
+        if os.path.basename(
+                os.path.normpath(include_dir)) == "include-fixed":
+            continue
+        if contains_intrinsic_headers(include_dir):
+            continue
+        result.append(include_dir)
+    return result
+
+
+def process_includes(includes_data):
+    start_mark = "#include <...> search starts here:"
+    end_mark = "End of search list."
+
+    for compiler in includes_data:
+        err = includes_data[compiler]
+        include_paths = []
+
+        do_append = False
+        for line in err.splitlines():
+            if line.startswith(end_mark):
+                break
+            if do_append:
+                line = line.strip()
+                fpos = line.find("(framework directory)")
+                if fpos == -1:
+                    include_paths.append(line)
+                else:
+                    include_paths.append(line[:fpos - 1])
+
+            if line.startswith(start_mark):
+                do_append = True
+
+        includes_data[compiler] = filter_includes(include_paths)
+
+
+def process_targets(target_data):
+    target_label = "Target:"
+
+    for compiler in target_data:
+        err = target_data[compiler]
+        target_clean = ""
+
+        for line in err.splitlines(True):
+            line = line.strip().split()
+            if len(line) > 1 and line[0] == target_label:
+                target_clean = line[1]
+
+        target_data[compiler] = target_clean
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+            description="Unify compiler target and includes files into one "
+                        "info file to be used with new CodeChecker versions.")
+    parser.add_argument('dir', metavar='DIR', help='Path to directory where '
+                        '"compiler_[includes/target].json" files are located.')
+    parser.add_argument('--clean', action='store_true', required=False,
+                        default=False, help='remove old '
+                        '"compiler_[includes/target].json" files')
+    args = parser.parse_args()
+
+    if not os.path.isdir(args.dir):
+        sys.stderr.write("%s is not a directory\n" % args.dir)
+        sys.exit(1)
+
+    includes_file = os.path.join(args.dir, 'compiler_includes.json')
+    if not os.path.isfile(includes_file):
+        sys.stderr.write("Cannot find 'compiler_includes.json' file in %s\n"
+                         % args.dir)
+        sys.exit(2)
+
+    target_file = os.path.join(args.dir, 'compiler_target.json')
+    if not os.path.isfile(target_file):
+        sys.stderr.write("Cannot find 'compiler_target.json' file in %s\n"
+                         % args.dir)
+        sys.exit(3)
+
+    with open(includes_file) as src:
```

Comment:
Please use io.open to make it easier to port to python3, and user unicode type for text everywhere in the new script.

Reply:
Done. Could you pls check?